### PR TITLE
Improve TS metrics docs

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

**Review by rachfop — APPROVED** [review]

**Comment by lorensr** [comment]
@rachfop your [1c82634](https://github.com/temporalio/documentation/pull/1024/commits/1c82634c6c5a5df2c036166de396b2017e279bdf) edit feels to me like the "Replace this" example here:

https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words

I guess the direct equivalent of "With this" example would be like "Interested? Read this." But I think the "Interested?" doesn't need to be asked—it's inherent in the fact that the reader is reading the Metrics section of a documentation page.

Also on the concision theme, I like that:

```
Deploy Checklist > Metrics
```

is shorter than:

```
Metrics section on the Deploy Checklist page
```

The `Page > Section` structure feels clear to me, and mirrors the breadcrumb:

image

Do you think it might be unclear to the reader? Perhaps:

- [Deploy Checklist ▶️ Metrics](https://docs.temporal.io/docs/typescript/production-deploy#logging-and-metrics)
- [Deploy Checklist ➡️ Metrics](https://docs.temporal.io/docs/typescript/production-deploy#logging-and-metrics)
- [Deploy Checklist 👉 Metrics](https://docs.temporal.io/docs/typescript/production-deploy#logging-and-metrics)
- Inline bubbles: (would have to code)
image

**Comment by rachfop** [comment]
> @rachfop your [1c82634](https://github.com/temporalio/documentation/pull/1024/commits/1c82634c6c5a5df2c036166de396b2017e279bdf) edit feels to me like the "Replace this" example here:
> 
> https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words
> 
> I guess the direct equivalent of "With this" example would be like "Interested? Read this." But I think the "Interested?" doesn't need to be asked—it's inherent in the fact that the reader is reading the Metrics section of a documentation page.

Fair point, I was adjusting the text based on the following:
The [Temporal Docs rely on Google's style](https://github.com/temporalio/documentation/blob/master/STYLE.md) (and Microsoft's when there isn't a suggstion from Google). Google's [Link text](https://developers.google.com/style/link-text) suggest to *use descriptive phrases that provide context for the material that you're linking to*.

> The Page > Section structure feels clear to me, and mirrors the breadcrumb:

I might be wrong, but I haven't seen that Breadcrumb styling in other sections of the docs and I may have missed the oringial intention of your choice to show the path to that section of the docs (which is why I suggested the revision). But it looks fun and I'll look more into the stylistic choice!

File: docs/typescript/production-deploy.md [hunk 1]
```diff
@@ -2,16 +2,21 @@
...
 ---
 
-Here is a non-exhaustive list of things we recommend doing before you deploy your Temporal app to production.
+The following are recommended steps to take before deploying your Temporal application to production.
 
-Note that this is separate from maintaining a production self-hosted Temporal **Cluster**, which has [its own checklist](https://docs.temporal.io/docs/server/production-deployment#checklist-for-scaling-temporal).
+## Production Temporal Cluster
 
-## Linting and Types
+Either use [Temporal Cloud](https://us17.list-manage.com/survey?u=2334a0f23e55fd1840613755d&id=f1895b6f4a) (apologies for the waitlist—it's not yet automated enough to support a large number of self-service customers) or deploy a self-hosted Temporal Cluster:
```
Comment (flossypurse) [inline docs/typescript/production-deploy.md:12]:
```suggestion
Either use [Temporal Cloud](https://us17.list-manage.com/survey?u=2334a0f23e55fd1840613755d&id=f1895b6f4a) (Join the waitlist for Temporal Cloud) or deploy a self-hosted Temporal Cluster:
```

Nix the apology I think.

Comment (lorensr) [reply docs/typescript/production-deploy.md:12]:
```suggestion
Either use Temporal Cloud ([join the waitlist](https://us17.list-manage.com/survey?u=2334a0f23e55fd1840613755d&id=f1895b6f4a)) or deploy a self-hosted Temporal Cluster:
```

**Comment by lorensr** [comment]
> Fair point, I was adjusting the text based on the following:
The [Temporal Docs rely on Google's style](https://github.com/temporalio/documentation/blob/master/STYLE.md) (and Microsoft's when there isn't a suggstion from Google). Google's [Link text](https://developers.google.com/style/link-text) suggest to use descriptive phrases that provide context for the material that you're linking to.

I see, thanks for the link. I think the "For more information, see link" guideline seems to not fit in this specific case, because it's really "For *any* information, see link" 😄

> I might be wrong, but I haven't seen that Breadcrumb styling in other sections of the docs and I may have missed the oringial intention of your choice to show the path to that section of the docs (which is why I suggested the revision). But it looks fun and I'll look more into the stylistic choice!

Intention is both concision and showing path, to help people remember where to find the info in future. Don't know whether this is done elsewhere in the docs. I wound up using it a lot in my book.

File: docs/typescript/logging.md [hunk 2]
```diff
@@ -151,9 +151,9 @@ const logger = winston.createLogger({
 await Core.install({ logger });
 ```
 
-## Monitoring SDK metrics
+## Metrics
 
-We are in the process of building out our SDK metrics capabilities. For now, please observe standard monitoring practices on your Workers in production (CPU, memory utilization, health checks).
+For more information about metrics, see the [Metrics section on the Deploy Checklist](/docs/typescript/production-deploy#metrics) page.
```
Comment (lorensr) [inline docs/typescript/logging.md:156]:
```suggestion
For information about metrics, see the [Metrics section on the Deploy Checklist](/docs/typescript/production-deploy#metrics) page.
```

**Pull request merged**